src/scss/theme/_book.scss

```diff
@@ -38,14 +38,15 @@
         top: 50px;
         left: 50%;
         z-index: 5;
-        transform: translateZ(28.5px) translateX(-50%);
+        -webkit-transform: translateZ(28.5px) translateX(-50%);
```

Comment:
Do you need to specify multiple rules instead of replacing them?

Reply:
Ok, rule applied for mobile device.

Reply:
No, why not have both rules? My understanding is that if you are going to have only 1 rule, to specify the basic one `transform`.

```
transform: ...
-webkit-transform: ...
```